docs-markdown/viz.md

```diff
@@ -0,0 +1,284 @@
+**ClusteredCircleViz**(_data, color_stops=None, radius_stops=None, cluster_radius=30, cluster_maxzoom=14, *args, **kwargs_)
+
+Parameter | Description
+--|--
+data | name of GeoJson file or object
+color_property | property to determine circle color
+color_stops | property to determine circle color
+radius_property | property to determine circle radius
+radius_stops | property to determine circle radius
+
+### Methods
+**ClusteredCircleViz.add_unique_template_variables**(_self, options_)
+Update map template variables specific to a clustered circle visual
+
+[View options](https://github.com/mapbox/mapboxgl-jupyter/blob/add-markdown-docs/docs-markdown/viz.md#params-4)
+
+### Usage
+```python
+import pysal.esda.mapclassify as mapclassify
+import pandas as pd
+import os
+from mapboxgl.utils import *
+from mapboxgl.viz import *
+
+# create pandas df from csv file
+df = pd.read_csv('points.csv')
+
+# Must be a public token, starting with `pk`
+token = os.getenv('MAPBOX_ACCESS_TOKEN')
+
+# Create a geojson file export from a Pandas dataframe
+df_to_geojson(df, filename='points.geojson',
+              properties=['Avg Medicare Payments', 'Avg Covered Charges', 'date'],
+                     lat='lat', lon='lon', precision=3)
+
+#Create a clustered circle map
+color_stops = create_color_stops([1,10,50,100], colors='BrBG')
+
+viz = ClusteredCircleViz('points.geojson',
+                          access_token=token,
+                          color_stops = color_stops,
+                          radius_stops = [[1,5], [10, 10], [50, 15], [100, 20]],
+                          cluster_maxzoom = 10,
+                          cluster_radius = 30,
+                          opacity = 0.9,
+                          center = (-95, 40),
+                          zoom = 3
+                        )
+
+viz.show()
+```
+![screen shot 2018-02-21 at 3 33 46 pm](https://user-images.githubusercontent.com/11286381/36511731-a4ca9af6-171c-11e8-9cde-60ef59e7e3b9.png)
+
+
+## class GraduatedCircleViz
+
+The `GraduatedCircleViz` object handles the creation of a graduated map and is built on top of the `MapViz` class.
+
+### Params
+**GraduatedCircleViz**(_data, label_property=None, color_property=None, color_stops=None, color_default='grey', color_function_type='interpolate', radius_property=None, radius_stops=None, radius_default=None, radius_function_type='interpolate', *args, **kwargs_)
+
+Parameter | Description
+--|--
+data | name of GeoJson file or object
+label_property | property to use for marker label
+color_property | property to determine circle color
+color_stops | property to determine circle color
+color_default | property to determine default circle color if match lookup fails
+color_function_type | property to determine `type` used by Mapbox to assign color
+radius_property | property to determine circle radius
+radius_stops | property to determine circle radius
+radius_default | property to determine default circle radius if match lookup fails
```

Comment:
Change to `radius of the circle to use if no lookup value matches the property value.  Only used for the **match** radius_function_type.`